packages/babel-helper-create-class-features-plugin/src/index.js

```diff
@@ -27,61 +25,23 @@ export { enableFeature, FEATURES, setLoose };
 const version = pkg.version.split(".").reduce((v, x) => v * 1e5 + +x, 0);
 const versionKey = "@babel/plugin-class-features/version";
 
-const getFeatureOptions = (options, name) => {
-  const value = options[name];
-
-  if (value === undefined || value === false) return { enabled: false };
-  if (value === true) return { enabled: true, loose: false };
-
-  if (typeof value === "object") {
-    if (
-      typeof value.loose !== "undefined" &&
-      typeof value.loose !== "boolean"
-    ) {
-      throw new Error(`.${name}.loose must be a boolean or undefined.`);
-    }
-
-    return { enabled: true, loose: !!value.loose };
-  }
-
-  throw new Error(
-    `.${name} must be a boolean, an object with a 'loose'` +
-      ` property or undefined.`,
-  );
-};
-
-export default declare((api, options) => {
-  api.assertVersion(7);
-
-  const fields = getFeatureOptions(options, "fields");
-  const privateMethods = getFeatureOptions(options, "privateMethods");
-  const decorators = getFeatureOptions(options, "decorators");
-
+export function createClassFeaturePlugin({
+  name,
+  feature,
+  loose,
+  manipulateOptions,
+}) {
   return {
-    name: "class-features",
+    name: "class-features/" + name,
```

Comment:
I think I'd rather keep the names consistent with the other plugins. Maybe we can delete this line and have the plugin do:
```
return {
  name: "...",
  createClassFeaturePlugin({ /* ... */ })
};
```
?